coalib/bearlib/languages/documentation/DocumentationComment.py

```diff
@@ -239,8 +247,9 @@ def from_metadata(cls, doccomment, docstyle_definition,
             assembled_doc += ''.join(section_desc)
 
         return DocumentationComment(assembled_doc, docstyle_definition, indent,
-                                    marker, range)
+                                    marker, position)
 
+    @lru_cache(maxsize=1)
```

Comment:
we don't reset the cache now, so once `assemble` is executed, the result won't change any more. Are you planning to reset the cache when properties change?